plans/benchmarks/benchmarks.go

```diff
@@ -284,14 +287,27 @@ func SubtreeBench(runenv *runtime.RunEnv) error {
 			return err
 		}
 
+		_, err = writer.SignalEntry(ctx, end)
+		if err != nil {
+			return err
+		}
+
+		// wait for everyone to be done; this is necessary because the sync
+		// service applies TTL by electing the first 5 publishers on the Redis
+		// Stream to own the keepalive. In this case, all 5 publishers will be
+		// THE publisher. In an ordinary test case, each instance will write a
+		// key and therefore the ownership will be distributed. That does not
+		// happen here, as all key publishing is concentrated on the publisher.
+		<-watcher.Barrier(ctx, end, int64(runenv.TestGroupInstanceCount))
+
 	case "receive":
 		runenv.RecordMessage("i am a subscriber")
 
 		// if we are receiving, wait for the publisher to be done.
 		<-watcher.Barrier(ctx, handoff, int64(1))
 
 		for _, tst := range tests {
-			ch := make(chan []byte, 1)
+			ch := make(chan *string, 100)
```

Comment:
Why 100? Ideally we don't want to consume more than necessary, and want to block the producer (in this case `watcher.Subscribe`.

I think 1 is fine.